### PR TITLE
Any to any framework for Syndeo Template

### PR DESCRIPTION
**Pull request by pabloem**
Hi @manavgarg  - can you start doing a code review please? Feel free to ask me any questions.

I still have to improve the test result output so we can read it better.

### CODE REVIEW

**Comment by manavgarg** [comment]
at a high level, the changes looks good. will review again once you are done.
Also, adding @xianhualiu and @pranavbhandari24 who might find this relevant.

File: syndeo-template/src/test/java/com/google/cloud/syndeo/perf/AnyToAnySyndeoTestLT.java [hunk 1]
```diff
@@ -0,0 +1,505 @@
...
+            CONFIGURATIONS.get(CONFIGURATION).getNumRows(),
+            "runtimeSeconds",
+            CONFIGURATIONS.get(CONFIGURATION).getRuntime().getStandardSeconds()));
+  }
+
+  @BeforeClass
+  public void setUpSourceAndSink() {}
+
+  @Test
+  public void testSelectedSinkAndSelectedSource() throws JsonProcessingException {
```
Comment (xianhualiu) [inline syndeo-template/src/test/java/com/google/cloud/syndeo/perf/AnyToAnySyndeoTestLT.java:382]: 1. suggest change name to testRadmonlySelectedSourceandSink.
2. add another test to test with all possible source-sink combinations with two for loops. This test is convenient for regression rest.

Comment (pabloem) [reply syndeo-template/src/test/java/com/google/cloud/syndeo/perf/AnyToAnySyndeoTestLT.java:382]: Can I add those changes as a follow-up to this one this week?

**Comment by bvolpato** [comment]
@pabloem Can you please squash the commits?

**Comment by pabloem** [comment]
yesssirrr!

File: syndeo-template/src/test/java/com/google/cloud/syndeo/perf/AnyToAnySyndeoTestLT.java (hunk 1 above)
Comment (xianhualiu) [reply syndeo-template/src/test/java/com/google/cloud/syndeo/perf/AnyToAnySyndeoTestLT.java:382]: 👍

**Pull request merged**